examples/interaction/js_callbacks/customjs_for_selection.py

```diff
@@ -1,3 +1,16 @@
+''' An interactive plot showcasing Bokeh's ability to add interactions
+using Custom Javascript.
+This example demonstrates selecting highlighted points on a graph.
+These points will show on the secondary graph which uses ``CustomJS``
+callbacks to update its plot.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.plotting.figure.circle, bokeh.layouts.row, bokeh.model.Model.js_on_change, bokeh.models.sources.ColumnDataSource, bokeh.models.callbacks.CustomJS
```

Comment:
similar observation as above
```suggestion
    :apis: bokeh.model.Model.js_on_change, bokeh.models.sources.ColumnDataSource, bokeh.models.callbacks.CustomJS
```